native/src/test/scala/sqs4s/internal/MessageSpec.scala

```diff
@@ -0,0 +1,81 @@
+package sqs4s.internal
+
+import java.time.Instant
+import java.util.concurrent.TimeUnit
+
+import fs2._
+import cats.effect.{Clock, IO}
+import cats.implicits._
+import org.http4s.Uri
+import org.http4s.client.blaze.BlazeClientBuilder
+import sqs4s.api._
+import sqs4s.api.lo.{ReceiveMessage, SendMessage}
+import sqs4s.internal.aws4.IOSpec
+import sqs4s.serialization.{MessageDecoder, MessageEncoder}
+
+import scala.concurrent.duration._
+
+class MessageSpec extends IOSpec {
+  override implicit lazy val testClock: Clock[IO] = new Clock[IO] {
+    def realTime(unit: TimeUnit): IO[Long] = IO.delay {
+      Instant.now().toEpochMilli
+    }
+
+    def monotonic(unit: TimeUnit): IO[Long] = ???
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Usage of ??? operator](https://app.codacy.com/manual/d2a4u/sqs4s/pullRequest?prid=4059007)